doc/sql_guide.md

```diff
@@ -0,0 +1,398 @@
+    
+# 3. Regression Test Deployment  
+## 3.1 Deployment overview    
+  No|role|user|ip|hostname|QUEUE_NAME|run_name
+  :---:|:--:|:---:|:---:|:---:|:---:|:---:
+  1|SQL/MEDIUM(Linux)|sql1|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+  2|SQL/MEDIUM(Linux)|sql2|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+  3|SQL/MEDIUM(Linux)|sql3|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+  4|SQL_BY_CCI|sqlbycci|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_CCI_LINUX_GIT|run_sql_by_cci
+  5|SQL (Linux, Standalone)|sql|192.168.1.77|func02|QUEUE_CUBRID_QA_SQL_PERF_LINUX|run_sql
+  6|SQL/MEDIUM (Windows)|qa|192.168.1.161|winfunc01|QUEUE_CUBRID_QA_SQL_WIN64|run_sql 
+## 3.2 Installation 
+* [Install CTP as Regression Test platform](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform) 
+* Configure tests
+  * SQL and MEDIUM configuration  
+  Create configuration file `CTP/conf/sql_local.conf`.  Then set same content as `CTP/conf/sql.conf` in [SQL Test](#21-sql-test).
+  * SQL_BY_CCI  
+  Create configuration file `CTP/conf/sql_by_cci_template.conf`.  Then set same content as `CTP/conf/sql_by_cci.conf` in [SQL_BY_CCI Test](#23-sql_by_cci-test).
+* Check out test cases
+  ```
+  cd ~/
+  git clone https://github.com/CUBRID/cubrid-testcases.git  
+  ```
+* Modify .bash_profile
+  ```
+  export CTP_HOME=$HOME/CTP
+  export CTP_SKIP_UPDATE=0
+  export CTP_BRANCH_NAME=develop
+  . ~/.cubrid.sh
+  export PATH=$HOME/CTP/bin:$HOME/CTP/common/script:$PATH:$HOME/.local/bin:$PATH:$HOME/bin
+
+  export TZ='Asia/Seoul'
+  export LC_ALL=en_US
+  ```
+* Add quick start script file    
+  * Create file `~/start_test.sh` with below content: 
+      ```
+      stop_consumer.sh 
+      prefix=`date "+%Y%m%d%H%M%S"`
+      cp nohup.out nohup.out.$prefix
+      echo "" > nohup.out
+      nohup start_consumer.sh -q QUEUE_CUBRID_QA_SQL_LINUX_GIT -exec run_sql &
+      ``` 
+  * sh start_test.sh  
+
+* Send test message   
+  Login message@192.168.1.91 and send test message like:  
+  * Test with debug build on Linux
+    ```
+    sender.sh QUEUE_CUBRID_QA_SQL_LINUX_GIT _build_url_ sql_debug default
+    sender.sh QUEUE_CUBRID_QA_SQL_LINUX_GIT _build_url_ medium_debug default
+    ```
+  * Test message with release build on Linux
+    ```
+    sender.sh QUEUE_CUBRID_QA_SQL_PERF_LINUX _build_url_ sql default
+    sender.sh QUEUE_CUBRID_QA_SQL_PERF_LINUX _build_url_ medium default
+    sender.sh QUEUE_CUBRID_QA_SQL_CCI_LINUX_GIT _build_url_ sql_by_cci default
+    ```
+  * Test message on Windows
+    ```
+    sender.sh QUEUE_CUBRID_QA_SQL_WIN64 _build_url_ sql default
+    sender.sh QUEUE_CUBRID_QA_SQL_WIN64 _build_url_ medium default
+    ```
+   _build_url_: provide a build URL like `http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8369-5a75e41/drop/CUBRID-10.2.0.8369-5a75e41-Linux.x86_64-debug.sh`.  
+   
+   
+# 4. Regression Test Sustaining  
```

Comment:
Follow these categories:

# 4. Regression Test Sustaining

## 4.1 Execute Test Manually

(move "Send test message" in https://github.com/CUBRID/cubrid-testtools/blob/16ae17ea25fce2784e2067568379cdc77ca2dcdb/doc/sql_guide.md#32-installation to here)

## 4.2 Verify Test Results

(follow this style:
https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/jdbc_guide.md#52-verify-test-results)

## 4.2 Code Coverage Test
(how to send test messages? how to get results in QA homepage?, etc)